### PR TITLE
fix skills logs, add support for raw logs, and better error reporting

### PR DESCRIPTION
**Pull request by pkandarpa-cs**
# Checklist:
Please check you fulfill ALL of the relevant checkboxes
- [ ] Notified docs of any potential USER-facing changes
- [ ] Added short description of the change - with relevant motivation and context. 
- [ ] Branch has the ticket number in its name (along with a ticket summary)
- [ ] Commented the code, particularly in hard-to-understand areas
- [ ] Added tests that prove my fix is effective or that my feature works
- [ ] Ran `npm test` and it passes
- [ ] Changes generate no new warnings
- [ ] Changes are backward compatible with older clusters

### CODE REVIEW

**Comment by pkandarpa-cs** [comment]
Sample output now:

```
+ cortex skills logs op opp-action
Failed to List Skill/Action Logs op/opp-action: Skill definition with name op not found
```
```
+ cortex skills logs opp opp-actio
Failed to List Skill/Action Logs opp/opp-actio: Action with name opp-actio not found in Skill with name opp. Available actions are opp-action
```
```
+ cortex skills logs opp opp-action
[{"name":"test-me-193d5-opp-opp-action-76679b8c79-kzwkc","logs":"INFO:     Started server process [1]\nINFO:     Waiting for application startup.\nINFO:     Application startup complete.\nINFO:     Uvicorn running on http://0.0.0.0:5000 (Press CTRL+C to quit)\n"}]
```
```
+ cortex skills logs opp opp-actio --raw
Failed to raw List Skill/Action Logs opp/opp-actio: Action with name opp-actio not found in Skill with name opp. Available actions are opp-action
```

```
+ cortex skills logs op opp-actio --raw
Failed to raw List Skill/Action Logs op/opp-actio: Skill definition with name op not found
```
```
+ cortex skills logs opp opp-action --raw
INFO:     Started server process [1]
INFO:     Waiting for application startup.
INFO:     Application startup complete.
INFO:     Uvicorn running on http://0.0.0.0:5000 (Press CTRL+C to quit)
```

**Comment by pkandarpa-cs** [comment]
This PR is dependent on https://github.com/CognitiveScale/cortex-catalog-service/pull/46 being merged in JFYI

File: src/client/catalog.js [hunk 1]
```diff
@@ -112,16 +112,18 @@ export default (class Catalog {
             .catch((err) => constructError(err));
     }
 
-    skillLogs(projectId, token, skillName, actionName, verbose = false) {
+    skillLogs(projectId, token, skillName, actionName, raw = false) {
```
Comment (jgielstra-cs) [inline src/client/catalog.js:115]: Could have used async syntax..  be getting rid of then().catch(). make code clearer...

File: src/client/catalog.js [hunk 2]
```diff
@@ -112,16 +112,18 @@ export default (class Catalog {
...
             headers: defaultHeaders(token),
-            searchParams: { verbose },
-        }).json()
-            .then((res) => ({ ...res }))
+            searchParams: { raw },
+        };
+        const gotResponse = raw ? got.get(endpoint, gotOptions).text() : got.get(endpoint, gotOptions).json();
+
+        return gotResponse
+            .then((res) => (raw ? res : { ...res }))
```
[the remark this answers is not in the document]
Comment (pkandarpa-cs) [reply src/client/catalog.js:126]: not sure.. it was always there. Its redundant seems like

**Pull request merged**